Library/Formula/node.rb

```diff
@@ -25,6 +25,7 @@ class Node < Formula
   option "without-completion", "npm bash completion will not be installed"
 
   depends_on :python => :build
+  depends_on :xcode => :build
```

Comment:
Node's gyp was updated in joyent/node@96dffb12177570a1311e97df501142e56b606c02 a bit over a year ago - the past several releases have had no issues with the CLT.